test/e2e/v1beta1/scripts/gh-actions/run-e2e-tune-api.py

```diff
@@ -36,7 +37,19 @@ def objective(parameters):
         "b": search.double(min=0.1, max=0.2)
     }
 
-    # [3] Create Katib Experiment with 4 Trials and 2 CPUs per Trial.
+    # [3] Create a dummy metric collector (DOES NOT HAVE A IMAGE)
+    metric_collector = V1Container(
+        name="dummy-collector",
+        image="dummy-collector:latest",
+        command=["python", "/app/dummy-collector.py"],
+        args=["--metric-name=result"],
+        env=[
+            client.V1EnvVar(name="EXPERIMENT_NAME", value=exp_name),
+            client.V1EnvVar(name="EXPERIMENT_NAMESPACE", value=exp_namespace)
+        ]
+    )
```

Comment:
I guess we can create another function `run_e2e_experiment_create_by_tune_custom` to run e2e tests for custom collector, rather than delete original e2e test for `StdOut` collector. Then we can run these e2e tests together in this file:)

WDYT👀 @prakhar479 @andreyvelich @tenzen-y @johnugeorge

Reply:
I have made corresponding changes to run both e2e tests (one with custom metrics collector and another with default metric collector [StdOut])